app/javascript/components/visualization/PlotDisplayControls.jsx

```diff
@@ -33,29 +50,83 @@ export default function RenderControls({ shownTab, exploreParams, updateExploreP
     distributionPointsValue = DISTRIBUTION_POINTS_OPTIONS[0]
   }
 
-  const showScatter = shownTab === 'scatter' &&
-                      (exploreParams.annotation.type === 'numeric' || exploreParams.genes.length)
+  const showScatter = shownTab === 'scatter'
+  const showColorScale = !!(showScatter && (exploreParams.annotation.type === 'numeric' || exploreParams.genes.length))
```

Comment:
Interesting --  '!!' is more familiar to me because it's the same in Ruby and JS.  But I could go either way.  @ehanna4  care to tiebreak?  :)

Reply:
Hmm, in general I think choosing the syntax that would be most language agnostic for our codebase (aka the things that works most in Ruby, JS, and Python) would be the best move. 

Of course the word 'Boolean' is likely recognizable across all languages.. but I think if '!!' works in both Ruby and JS I'm not convinced making a change to the PR is worth it. 

So in conclusion I think both would be perfectly acceptable but since it's already '!!' I don't think a change to 'Boolean()' provides enough benefit to outweigh the con of having to make the change.